### PR TITLE
Planning

### PR DESCRIPTION
**Pull request by vchadha**
## Changes
PROJECTION BOIS

## Type of Issue
FUN TIMES

## Reference Ticket
....

## Checklist

[//]: # (Make sure to fetch and rebase from upstream if needed)
[//]: # (Make sure to add any unit, integration, or functional tests needed)
[//]: # (If tests were added please add them to their corresponding test runner ie. unit_runner etc.)
[//]: # (Make sure to add any new dependencies to requirements files)

- [x] Tests pass

### CODE REVIEW

File: test/battletoads/planning/ASPlannerTest.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import static org.junit.Assert.*;
+import org.junit.Test;
+import org.junit.BeforeClass;
+
+import battlecode.common.MapLocation;
+import battlecode.common.RobotController;
+
+import battletoads.planning.ASPlanner;
+
+public class ASPlannerTest {
```
Comment (noah-brabec) [inline test/battletoads/planning/ASPlannerTest.java:12]: Java doc please

**Review by justinro-underscore — APPROVED** [review]

File: src/battletoads/planning/ASPlanner.java [hunk 2]
```diff
@@ -60,21 +60,14 @@ private void initialize() {
      * @param MapLocation to project
      * @return projected location
```
Comment (noah-brabec) [inline src/battletoads/planning/ASPlanner.java:61]: Author tag plase

**Review by noah-brabec — APPROVED** [review]
Overall, I don't really get it. But I like you well enough to aprovee and LGTM so send it

File: test/battletoads/planning/ASPlannerTest.java (hunk 1 above)
Comment (vchadha) [reply test/battletoads/planning/ASPlannerTest.java:12]: u r wrong

**Pull request merged**